### ISSUE TITLE
Remembered stream does not add listener when source is created directly via 'xs.create'

### ISSUE THREAD

**Issue by ntilwalli** [staltz/xstream#69]
I noticed this odd behavior (what looks like a bug).  When `remember` is called directly on a raw stream created via `xs.create` that has no producer (i.e. which will emit based on the `shamefully*` methods), it doesn't add listeners since the `_prod` property is `undefined`.  
```
import xs from 'xstream'

const test$ = xs.create()

// This line works
//const oddBehavior$ = test$  

// This line does not work. Why?
const oddBehavior$ = test$.remember()

oddBehavior$.addListener({
  next: n => bin.log(`Next: ${n}`),
  error: e => bin.log(`Error: ${e}`),
  complete: () => bin.log(`Complete`)
})

test$.shamefullySendNext(1)
```
You can see the issue here: http://www.webpackbin.com/4JFCBWrL-

**Comment by ntilwalli** [comment]
Convo discussion this issue on Gitter is here: https://gitter.im/cyclejs/core?at=577c364a4e50bf894a20066b

**Comment by staltz** [comment]
Fixed in v5.1.3
